packages/service-maker/src/pages/Auth/Login.tsx

```diff
@@ -1,81 +1,198 @@
+import { useCallback, useEffect } from 'react';
 import { useForm } from 'react-hook-form';
-import { ButtonMaker } from 'components/button/button';
-import { FormMaker } from 'components/form/form';
-import { InputMaker } from 'components/input/input';
-import { SideBarMaker } from 'components/sidebar/sidebar';
-import api from 'services/api';
+import { useNavigate } from 'react-router-dom';
+import { userActions } from 'services/api/routes/user';
+import { styled } from 'styled-components';
 
-type Inputs = {
+interface IFormInputs {
   email: string;
   password: string;
-};
+}
 
 export function Login() {
-  const { register, handleSubmit } = useForm<Inputs>();
-  const { mutateAsync: loginMutation } = api.user.useLogin();
-
-  const onSubmit = async (data: Inputs) => {
-    try {
-      await loginMutation({ loginUserModel: data });
-      window.location.href = '/home';
-    } catch (err) {
-      // pop up error
-    }
-  };
+  const navigate = useNavigate();
+  const { data } = userActions.useGetUser();
+  const { mutateAsync: loginMutation } = userActions.useLogin();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<IFormInputs>();
+
+  const onSubmit = useCallback(
+    (values: IFormInputs) => {
+      loginMutation({ loginUserModel: values }).then(() => {
+        navigate('/home');
+      });
+    },
+    [loginMutation, navigate],
+  );
 
-  const handleRegisterClick = () => {
-    window.location.href = 'register';
-  };
+  useEffect(() => {
+    if (data) {
+      // navigate('/home');
+    }
+  }, [data, navigate]);
```

Comment:
?

Reply:
The modification made is not really related to this issue and adds a bug. I'd prefer to deactivate it while we sort it out with BACK and FRONT